lib/ddtrace.rb

```diff
@@ -75,10 +74,12 @@ module Datadog
 require 'ddtrace/contrib/redis/integration'
 require 'ddtrace/contrib/resque/integration'
 require 'ddtrace/contrib/rest_client/integration'
-require 'ddtrace/contrib/rspec/integration'
 require 'ddtrace/contrib/sequel/integration'
 require 'ddtrace/contrib/shoryuken/integration'
 require 'ddtrace/contrib/sidekiq/integration'
 require 'ddtrace/contrib/sinatra/integration'
 require 'ddtrace/contrib/sneakers/integration'
 require 'ddtrace/contrib/sucker_punch/integration'
+
+# Load extensions from CI namespace
+require 'datadog/ci'
```

Comment:
There is this public repo https://github.com/DataDog/datadog-ci where we have a command but it's more used for synthetics and sourcemaps. I think that makes this a bit confusing maybe you could namespace it under `datadog/ciapp` or `ddtrace/contrib/ciapp`

Reply:
That's a JS repo. Is there an equivalent Ruby repo?

Reply:
No, there is no equivalent in ruby, but this is a CLI that can be installed by users from any language.

Reply:
If it's not in Ruby, and we don't plan to port it to Ruby, then I don't see any conflict. There's also no reason that if it were, that they couldn't be made to coexist inside the same CI namespace.

Reply:
I thiink it should be ok to have `datadog/ci`, I don't see folks getting confused by this in ruby.